sale_order_partial_advance/README.rst

```diff
@@ -0,0 +1,44 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+Sale Order Partial Advance
+===========================
+
+This module was written to extend the sale advance management. It allows to
+use an advance amount progressively when you invoices only some order lines.
+
+Example:
```

Comment:
Your example is a good initiative. I have some doubts though and propose a rephrase as followed (but let me know if I understood it properly). 
You create a sale order with 4 different products for a total amount of 2000.
The customer pays an advance of 800. All the products will not be delivered 
at once and when you invoice the first delivery of 500,
you want to keep a part of the advance as warranty and only deduct 200
out of the 800 paid.